deploy/charts/rook-ceph/templates/configmap.yaml

```diff
@@ -27,12 +27,18 @@ data:
 {{- if .Values.csi.cephFSPluginUpdateStrategy }}
   CSI_CEPHFS_PLUGIN_UPDATE_STRATEGY: {{ .Values.csi.cephFSPluginUpdateStrategy | quote }}
 {{- end }}
+{{- if .Values.csi.cephNFSPluginUpdateStrategy }}
```

Comment:
set these values only if nfs is enabled?

Reply:
> set these values only if nfs is enabled?

The value will anyway not be used, I think we can leave it as it is, instead of adding verbosity.